.ci/install_pypy36.py

```diff
@@ -10,7 +10,7 @@
     raise SystemExit
 
 plat = {"darwin": "osx64", "linux": "linux64"}[sys.platform]
-filename = "pypy3.6-v7.3.1-{}.tar.bz2".format(plat)
+filename = f"pypy3.6-v7.3.1-{plat}.tar.bz2"
```

Comment:
Is this file still needed? GHA has PyPy support built in, as does cibuildwheel.

Reply:
Probably not.